tests/units/modules/clustering/kubevirt/test_kubevirt_raw.py

```diff
@@ -0,0 +1,66 @@
+import json
+import sys
+import pytest
+
+from ansible.compat.tests.mock import patch, MagicMock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+
+from openshift.dynamic import Resource
+
+# FIXME: paths/imports should be fixed before submitting a PR to Ansible
+sys.path.append('lib/ansible/modules/clustering/kubevirt')
+
+import kubevirt_raw as mymodule
+
+
+def set_module_args(args):
+    args = json.dumps({'ANSIBLE_MODULE_ARGS': args})
+    basic._ANSIBLE_ARGS = to_bytes(args)
+
+
+class AnsibleExitJson(Exception):
+    """Exception class to be raised by module.exit_json and caught
+    by the test case"""
+    pass
+
+
+def exit_json(*args, **kwargs):
+    if 'changed' not in kwargs:
+        kwargs['changed'] = False
+    raise AnsibleExitJson(kwargs)
+
+
+class TestKubeVirtRawModule(object):
```

Comment:
I think that the class and two methods could be moved to utility module which could be use across tests.

Reply:
The glue code is copy&pasted from the two other existing test files. I'd prefer to finish this code as is, get it merged, then have a separate PR for extracting any commonalities across all the test code.

Reply:
We should not copy the code. This is perfect place to push a new PR which would make the code common and we can reuse it here.

Reply:
Ok, that order it will be then.